internal/apm/command_deployment.go

```diff
@@ -39,8 +40,13 @@ The list command returns deployments for a New Relic APM application.
 `,
 	Example: "newrelic apm deployment list --applicationId <appID>",
 	Run: func(cmd *cobra.Command, args []string) {
+		if apmAppID == 0 {
```

Comment:
I wonder if we can just `cmdDeploymentList.MarkFlagRequired("applicationID")`?  Even though it's a persistent flag?

Reply:
You can't since it's a persisted flag, and while there is a MarkPersistentFlagRequired func, reading through the docs / issues say it was never actually implemented to enforce a flag as required 😢